### PR TITLE
Item caching fix

### PR DESCRIPTION
**Pull request by hategan**
Fixes whole-tale/girder_wt_data_manager#39

### CODE REVIEW

File: girderfs/core.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 from queue import Queue
 import traceback
 
-logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.WARNING)
+logging.basicConfig(format='%(asctime)-15s %(levelname)s:%(message)s', level=logging.DEBUG)
```
Comment (Xarthisius) [inline girderfs/core.py]: I'd keep logging level set to WARNING in production/master.

**Comment by Xarthisius** [comment]
Fixes the issue I was seeing! Thanks @hategan!

File: girderfs/core.py (hunk 1 above)
Comment (hategan) [reply girderfs/core.py]: By all means. I didn't realize I was changing the level. I thing a good way to deal with this in a general way would be to add a command line option.

Comment (hategan) [reply girderfs/core.py]: s/thing/think/

**Pull request merged**